TailorTests/Shared/TestSubjects.swift

```diff
@@ -78,8 +78,20 @@ struct TestPersonStruct: Inspectable, Mappable, Equatable {
 
     birthDate <- map.transform("birth_date", transformer: dateTransformer)
 
-    relatives <- map.objects("relatives")
-    job <- map.object("job")
+    relatives <- map.relations("relatives")
+    job <- map.relation("job")
+  }
+}
+
+struct TestImmutable: MappableOnce {
+  var firstName: String
+  var lastName: String
+
+  init(_ map: JSONDictionary) throws {
+    do {
+      firstName = try <-map.property("firstName")
```

Comment:
Would you add a space here, like: `firstName = try <- map.property("firstName")`?